tools/ds-identify

```diff
@@ -1586,6 +1586,28 @@ dscheck_VMware() {
     return "${DS_NOT_FOUND}"
 }
 
+WSL_cloudinit_dir_in() {
+    _RET=""
+    for m in "$@"; do
+        cmdexe="$m/Windows/System23/cmd.exe"
```

Comment:
we should probably declare the cmdexe variable as local to `WSL_cloudinit_dir_in` using the `local` keyword at the top of this function.